### DISCUSSION TITLE
Passed the teacher certification but Copilot do not have Pro access.

### DISCUSSION THREAD

**Discussion by ruirui-zhao** [community/community#147743] [GitHub Education]
### Hi everyone,

I have already passed the teacher certification, but my Copilot still shows that I do not have Pro access. What should I do?

**Comment by IrviWade** [comment]
Hey! 🤔 You can try through : https://github.com/github-copilot/free_signup ^^

> ### Hi everyone,
> I have already passed the teacher certification, but my Copilot still shows that I do not have Pro access. What should I do?
